api/src/main/java/org/openmrs/module/reporting/dataset/definition/evaluator/IterableDataSetEvaluator.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.reporting.dataset.definition.evaluator;
+
+import org.openmrs.module.reporting.ReportingException;
+import org.openmrs.module.reporting.dataset.DataSet;
+import org.openmrs.module.reporting.dataset.definition.DataSetDefinition;
+import org.openmrs.module.reporting.definition.evaluator.DefinitionEvaluator;
+import org.openmrs.module.reporting.evaluation.EvaluationContext;
+import org.openmrs.module.reporting.evaluation.EvaluationException;
+
+/**
+ * This subinterface defines the methods needed to allow an evaluator to be wrapped in a
+ * {@link DataSet}, making it easy to implement the evaluator for many
+ * {@link DataSetDefinition}s.
+ */
+public interface IterableDataSetEvaluator extends DefinitionEvaluator<DataSetDefinition> {
+
+	public DataSet evaluate(DataSetDefinition dataSetDefinition, EvaluationContext evalContext) throws ReportingException, EvaluationException;
+	
+}
```

Comment:
Can you please describe why you created this new Interface?  How is this different from the existing DataSetEvaluator interface?  What value is this providing?  I don't see any new methods.  Is this just a marker interface?  Is that needed?